CHANGELOG.rst

```diff
@@ -4,6 +4,32 @@ Check_Point.gaia Release Notes
 
 .. contents:: Topics
 
+v6.0.0
```

Comment:
please make it 7.0.0 as someone else is releasing 6.0.0
and pay attention to version added in other files
update all the same